test/src/main/java/org/apache/accumulo/test/ScanServerGroupConfigurationIT.java

```diff
@@ -166,7 +166,7 @@ public void testClientConfiguration() throws Exception {
         // Bump the number of scan serves that can run to start the GROUP1 scan server
         getCluster().getConfig().setNumScanServers(2);
         getCluster()._exec(ScanServer.class, ServerType.SCAN_SERVER, Map.of(),
-            new String[] {"-g", "GROUP1"});
+            new String[] {"-o", "sserver.group=GROUP1"});
```

Comment:
Would there be a benefit of using the property name `SSERV_GROUP_NAME` here?

Reply:
Comment added in fa7bd35